carma/launch/guidance.launch

```diff
@@ -91,7 +91,7 @@
   <!-- Control Stack -->
   <!-- Waypoint Follower Config-->
   <node pkg="rostopic" type="rostopic" name="config_waypoint_follower_rostopic"
-        args="pub -l config/waypoint_follower autoware_config_msgs/ConfigWaypointFollower '{ header: auto, param_flag: 0, velocity: 5.0, lookahead_distance: 4.0, lookahead_ratio: 2.0, minimum_lookahead_distance: 6.0, displacement_threshold: 0.0, relative_angle_threshold: 0.0 }' "
+        args="pub -l config/waypoint_follower autoware_config_msgs/ConfigWaypointFollower '{ header: auto, param_flag: 0, velocity: 5.0, displacement_threshold: 0.0, relative_angle_threshold: 0.0 }' "
```

Comment:
If we take those three parameters out, is this line at all still necessary? Could we remove it?

Reply:
Pure pursuit is still reading the param flag and velocity values from here. The rest are not being used. Do we want to move those two to the calibration folder as well?

Reply:
You could move them or if these are not at all likely to change and they are the defaults just let them remain as default and remove the pub.